packages/connect-core/src/entities/Organization.ts

```diff
@@ -125,14 +154,83 @@ export default class Organization {
   }
 
   ///////// INTENTS ///////////
-  appIntent(
-    appAddress: Address,
-    functionName: string,
-    functionArgs: any[]
-  ): TransactionIntent {
-    return new TransactionIntent(
-      { contractAddress: appAddress, functionName, functionArgs },
-      this,
+
+  /**
+   * Calculate the transaction path for a transaction to an external `destination`
+   * (not the currently running app) that invokes a method matching the
+   * `methodAbiFragment` with `params`.
+   *
+   * @param  {string} destination Address of the external contract
+   * @param  {object} methodAbiFragment ABI fragment of method to invoke
+   * @param  {Array<*>} params
+   * @param  {Object} options
+   * @return {Promise<ForwardingPath>} An object that represents the forwarding path corresponding to an action.
+   *   If the destination is a non-installed contract, always results in a
+   *   single step transaction.
+   */
+  async intent(
+    destination: AppOrAddress,
+    methodAbiFragment: ethersUtils.FunctionFragment,
+    params: any[],
+    options?: PathOptions
+  ): Promise<ForwardingPath | undefined> {
+    const sender = options?.actAs || this.connection.actAs
+    if (!sender) {
+      throw new Error(
+        `No sender address specified. Use 'actAs' option or set one as default on your organization connection.`
+      )
+    }
+
+    const installedApps = await this.apps()
+
+    return organizationIntent(
+      sender,
+      destination,
+      methodAbiFragment,
+      params,
+      installedApps,
+      this.connection.ethersProvider
+    )
+  }
+
+  //////// DESCRIPTIONS /////////
+
+  // Return a description of the forwarding path encoded on the evm script
+  async describeScript(script: string): Promise<ForwardingPathDescription> {
+    const installedApps = await this.apps()
+
+    const describedSteps = await describePath(
+      decodeForwardingPath(script),
+      installedApps,
+      this.connection.ethersProvider
+    )
+
+    // TODO: Add decorators
+    // // Add name and identifier decoration
+    // const docoratedStep = describedSteps.map(async (step) => {
+    //   const app = installedApps.find((app) => app.address === step.to)
+
+    //   if (app) {
+    //     return {
+    //       ...step,
+    //       identifier: app.appId,
+    //       name: app.artifact.appName,
+    //     }
+    //   }
+
+    //   return step
+    // })
+
+    return new ForwardingPathDescription(describedSteps)
+  }
+
+  // Try to describe a single transaction using Radspec on the context of the organization
+  async describeTransaction(
```

Comment:
A new helper that allows describing a single transaction. I wonder if having it at the Organization is the best place? I think maybe is something we would like to provide on the `ForwardingPath` instead?